src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+</head>
+<body>
+  <div class="main">
+    <h1>Leaderboard</h1>
+  <div class="main-board">
+    <div id="display-scores">
+      <div id="displa-score-title"><h2>Recent Scores</h2><input type="button" value="Refresh"></div>
+      <ul>
+        <li><p>Name: 100</p></li>
+        <li><p>Name: 50</p></li>
+        <li><p>Name: 20</p></li>
+        <li><p>Name: 78</p></li>
+        <li><p>Name: 43</p></li>
+      </ul>
```

Comment:
- [ ] Great job so far :clap:  However You must use an array of objects to store these Scores then create a function to render through the array and create dynamically with the information from the object array as you did with your To-do list project.  (all the HTML of the section is created when the page loads).
- [ ] Don't forget to use ES6 modules to write modular JavaScript with proper ES6 syntax  and
- [ ] And  you should also use  `import` and `export` to import it as a module

eg: 

``` 
 const array = [
  { x: 'name', y: 100 },
 { x: 'name', y: 100 }
];


export const render = (arr) => {
  arr.forEach((game) => {
    ......
  });
};

```

Reply:
ok, I´ll do the changes, but I read the requirement for this request and it just say Write plain HTML markup with minimum styling, It doesn't say anything about using js for now. It would be better if that information were explicit in the requirements.

Reply:
Hi Mar, Thank you for the feedback 👍  I understand that it might be confusing, I'll reach out to the responsible people and ask if we can change anything to avoid this kind of confusion in the future.